package.json

```diff
@@ -19,14 +19,14 @@
   "version": "3.0.0",
   "private": true,
   "scripts": {
-    "dev": "cross-env PORT=9090 nuxt dev",
+    "dev": "cross-env PORT=9090 nuxi dev",
```

Comment:
Why is somewhere `nuxi` and somewhere `nuxt`?

Reply:
@vikiival https://v3.nuxtjs.org/bridge/overview#nuxi this might answer

btw you'll maybe need to run `nuxi prepare` for new Types

Reply:
currently using the latest version of nuxt-bridge (nuxi) not generated `dist` or `.output` folder when I tested it. so, for now, use `nuxt` based on this https://github.com/nuxt/bridge/issues/31#issuecomment-1097947361